web/components/file-upload-button.tsx

```diff
@@ -10,7 +10,11 @@ export function FileUploadButton(props: {
   const ref = useRef<HTMLInputElement>(null)
   return (
     <>
-      <button className={className} onClick={() => ref.current?.click()}>
+      <button
+        type={'button'}
```

Comment:
It was reloading the page when on the create dashboard form [https://stackoverflow.com/questions/1878264/how-do-i-make-an-html-button-not-reload-the-page]